sweagent/agent/models.py

```diff
@@ -236,6 +241,7 @@ class OpenAIModel(BaseModel):
         "gpt3-0125": "gpt-3.5-turbo-0125",
         "gpt4-turbo": "gpt-4-turbo-2024-04-09",
         "gpt4o": "gpt-4o-2024-05-13",
+        "gpt-4o-mini": "gpt-4o-mini-2024-07-18",
```

Comment:
I find it a bit inconsistent that all OpenAI GPT models so far are like `gpt3` or `gpt4o`, and now gpt-4o-mini introduces the dash between gpt and the version number.